.github/workflows/ci-go.yml

```diff
@@ -36,6 +39,9 @@ jobs:
         with:
           fetch-depth: 1
 
+      - name: 'Install gpgme'
+        run: sudo apt-get update && sudo apt-get install -y libgpgme-dev
+
```

Comment:
```suggestion
```
We don't need to setup this dependency with the `containers_image_openpgp` build tag